firebase-ml-modeldownloader/firebase-ml-modeldownloader.gradle

```
@@ -89,4 +89,17 @@ dependencies {
     //Android compatible version of Apache httpclient.
     testImplementation 'org.apache.httpcomponents:httpclient-android:4.3.5.1'
     testImplementation 'org.mockito:mockito-core:3.3.3'
-}
\ No newline at end of file
+
+    androidTestImplementation 'junit:junit:4.13-beta-2'
```

nit, looks like there is a stable release already available
```suggestion
    androidTestImplementation 'junit:junit:4.13.1'
```

---

Done